app/connectors/connectors/sql/src/main/resources/META-INF/syndesis/connector/sql.json

```diff
@@ -196,21 +196,21 @@
         },
         "propertyDefinitionSteps": [
           {
-            "name": "Procedure Name",
-            "description": "Select the Stored Procedure",
+            "name": "Procedure name",
+            "description": "Select the stored procedure.",
             "properties": {
               "procedureName": {
                 "kind": "property",
                 "group": "common",
-                "displayName": "Procedure Name",
+                "displayName": "Procedure name",
                 "required": false,
                 "type": "string",
                 "javaType": "java.lang.String",
                 "tags": [],
                 "deprecated": false,
                 "secret": true,
                 "componentProperty": true,
-                "description": "Name of the Store Procedure"
+                "description": "Name of the store procedure"
```

Comment:
Should that be "Name of the stored procedure" ? I not certain of the context.
I know this was approved and merged. Sorry if it is no longer good to comment here.

Reply:
Should these lines end in a period?
* Invoke SQL or stored procedures
* Invoke SQL to obtain, store, update, or delete data
* SQL SELECT statement to be executed
* Name of the store procedure
  * Per Tova's comment, "store" should be "stored"

And I might recommdend changing `i.e. ':#MYPARAMNAME'` to `(i.e., ':#MYPARAMNAME')` - parenthesis optional, but "i.e." should be followed by a comma.

Reply:
If it is a complete sentence, then there should be a period at the end. The imperative sentences above, the ones that start with "Invoke" should have periods. Are these tooltips?  The second two bullet items above are not complete sentences, so no period at the end. But, if these are all tooltips on the same page, then they should all have periods at the end. The rule that applies for that is:  In a list of things, if one list item is a complete sentence, then each list item should have a period at the end. I hope that makes sense. 
And I'll just say here that if we could implement the process for extracting tooltips from source files for user doc, then I think we would all save a lot of time. There's an issue for that somewhere. 
Finally, I don't know the context for your last comment, but "i.e." should not appear in anything the user sees. It's Latin and not English and problematic for translation. In this particular instance, you want either "for example," or "that is," I can't tell which would be correct.